app/controllers/users_controller.rb

```diff
@@ -56,4 +62,8 @@ def set_user
     @user = User.find_by_id(user_id)
   end
 
+  def valid_user_params?
+    User.new(user_params).valid?
```

Comment:
Good idea, but we're doing a lot of work here to only return a boolean (and we're only using it in one place!). Keep in mind that `.create` is `.new + .save`.